docs/API.md

```diff
@@ -616,6 +617,53 @@ minioClient.removeObject('mybucket', 'photo.jpg', function(err) {
 })
 ```
 
+<a name="removeObjects"></a>
+### removeObjects(bucketName, objectsList[, callback])
+
+Remove all objects in the objectsList.
+
+__Parameters__
+
+
+| Param | Type | Description |
+| ---- | ---- | ---- |
+| `bucketName` | _string_ | Name of the bucket. |
+| `objectsList`  | _object_  |  list of objects in the bucket.  |
```

Comment:
This can be changed to `list of objects in the bucket to be removed`

Reply:
changed it @kannappanr